api/integrations.go

```diff
@@ -40,6 +40,12 @@ const (
 	// AWS CloudTrail integration type
 	AwsCloudTrailIntegration
 
+	// AWS Gov Cloud Config integration type
+	AwsGovCloudCfgIntegration
+
+	// AWS GovCloud CloudTrail integration type
+	AwsGovCloudCTIntegration
```

Comment:
Shouldn't this be 🤔
```suggestion
	AwsGovCloudCloudTrailIntegration
```

Reply:
Depends on how you feel about CloudCloud. I went with CT because I didn't like the look of it.